internal/api/handlers/operations_handler.go

```diff
@@ -141,27 +143,55 @@ func (h *OperationsHandler) fromOperationsToResponses(entities []*operation.Oper
 }
 
 func (h *OperationsHandler) fromOperationToResponse(entity *operation.Operation) (*api.Operation, error) {
-	status, err := entity.Status.String()
+	operation := &api.Operation{
+		Id:             entity.ID,
+		DefinitionName: entity.DefinitionName,
+		SchedulerName:  entity.SchedulerName,
+		CreatedAt:      timestamppb.New(entity.CreatedAt),
+	}
+
+	var err error
+	operation.Status, err = entity.Status.String()
 	if err != nil {
 		return nil, fmt.Errorf("failed to convert operation entity to response: %w", err)
 	}
+
+	operation.ExecutionHistory = h.fromOperationEventsToResponse(entity.ExecutionHistory)
+	if err != nil {
```

Comment:
@arthur29 this error check here doesn't make sense since fromOperationEventsToResponse does not return any error, this if clause will never be reached since we are already checking this error above.